core/src/validator.rs

```diff
@@ -583,6 +583,18 @@ impl Validator {
                 info!("Hard forks: {:?}", hard_forks);
             }
         }
+        let pyth_keys = bank.get_pyth_keys();
+        for (key_name, pk_res) in pyth_keys {
+            match pk_res {
+                Ok(pk) => {
+                    info!("{}: {}", key_name, pk);
```

Comment:
It's good to prefix this with "Accumulator"

Reply:
fixed thanks